src/Equinor.ProCoSys.IPO.Query/GetInvitations/GetInvitationsQueryBase.cs

```diff
@@ -0,0 +1,202 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Equinor.ProCoSys.IPO.Domain;
+using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
+using Microsoft.EntityFrameworkCore;
+
+namespace Equinor.ProCoSys.IPO.Query.GetInvitations
+{
+    public abstract class GetInvitationsQueryBase
+    {
+        protected IQueryable<InvitationDto> CreateQueryableWithFilter(IReadOnlyContext context, string projectName, Filter filter, DateTime utcNow)
+        {
+            var startOfThisWeekUtc = DateTime.MinValue;
```

Comment:
So the business logic for when an invitation is Overdue in IPO follow the same rules as preservation?
In preservation it is OK to preserve within the week. It is ok to preserve on a Tuesday and on the Sunday. Same same.

So: If the IPO round is invited to happen on Tuesday at 14:30, is it overdue when that paricular week has ended (I.e 1 second after midnight at upcoming Sunday)